Drivers/Dxl/dxl.c

```diff
@@ -391,15 +405,15 @@ void dxl_request_manager(void)
                 {
                     int pos = (int)((1024 - 1) * ((300 / 2 + dxl[last].val) / 300));
                     servo_set_raw_word(dxl_table[i], SERVO_REGISTER_MIN_ANGLE, pos, DXL_TIMEOUT);
-                    HAL_Delay(1);
+                    HAL_Delay(500);
```

Comment:
I did not see that I added this into the commit. @nicolas-rabault Did you try if it works with 1ms? 500ms is definitely way too much here.

Reply:
No I don't think I test it, I don't have any motor to test it here. And I think we could have difference between motors models. But yes it's definitely way too big indeed.
1ms is also like an eternity for motors so I think it's enough for it to process the msg.

Reply:
I'll test it this afternoon and update with the correct version.

Reply:
1ms seems to work fine! The PR is ok on my side.